add-on/chrome/bootstrap.js

```diff
@@ -1039,6 +1039,14 @@ function loadDefaultPrefs() {
       }
     }
   });
+
+  if (Services.vc.compare(Services.appinfo.version, "47.0a1") < 0) {
+    branch.setBoolPref("loop.remote.autostart", false);
```

Comment:
We don't need to do this, because we also check if the current window is in e10s mode - in other words: when e10s is disabled in the browser, we will not boot conversation windows in e10s mode.
We follow the rules of the host.